components/FormCard.tsx

```diff
@@ -121,6 +123,7 @@ export const FormCard: React.FC<FormCardProps> = ({
           )}
         </div>
         <div className="text-left">{optionsList}</div>
+        {submitError && <h6 className="text-danger">{submitError}</h6>}
```

Comment:
For accessibility reason, it's best not to skip headings. If you need to use the `h6` styling. You could just use the class. 

https://webaim.org/techniques/semanticstructure/#contentstructure

Reply:
That is good to know but I would like to open a separate issue for that as it looks like there are alot of accessibliy issues on the page that should all get fixed together.

Current headers are 
- no `h1` header
- `h2` is the form lesson title 
- `h5` as the form labels (which are not linked to the inputs at all) 
- `h6` are error messages.

Biggest accessibility issue is gonna be the fact that the form labels are not labels or linked to the inputs in any way

Then the lesson selector nav bar is not keyboard accessible at all either